src/main/java/com/dev/cinema/dao/impl/CinemaHallDaoImpl.java

```diff
@@ -56,4 +60,16 @@ public CinemaHall add(CinemaHall cinemaHall) {
             throw new DataProcessingException("Can't show all cinema halls", e);
         }
     }
+
+    @Override
+    public CinemaHall getById(Long id) {
```

Comment:
I think here we can use just method get from hibernate

Reply:
ok